CONTRIBUTING.md

```diff
@@ -7,75 +7,66 @@ The following are a set of guidelines for contributing to the project.
 #### Contents
 
 * [Code of Conduct](#code-of-conduct)
-* [Ways of Contributing](#ways-of-contributing)
-  * [Joining Feature Discussions](#joining-feature-discussions-) 💭
-  * [Joining Developer Discussions](#joining-developer-discussions-) 💬
-  * [Reporting Bugs](#reporting-bugs-) 🐛
-  * [Requesting Features](#requesting-features-) 🚀
+* [Ways to Contribute](#ways-to-contribute)
+  * [Questions or Enhancement Requests](#questions-or-enhancement-requests-) 💭
+  * [Reporting and Fixing Bugs](#reporting-and-fixing-bugs-) 🐛
+  * [Submitting Features](#submitting-features-) 🚀
   * [Contributing Code](#contributing-code-) 👾
   * [Reviewing Code](#reviewing-code-) 👀
   * [Documenting](#documenting-) 📜
+  * [Joining Feature Discussions](#joining-feature-discussions-) 💭
+  * [Joining Developer Discussions](#joining-developer-discussions-) 💬
 * [Issue Report Guidelines](#issue-report-guidelines)
 * [Contributor License Agreement](#contributor-license-agreement)
 
 ## Code of Conduct
 
 This project adheres to the Adobe [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to the team.
 
-## Ways of Contributing
+## Ways to Contribute
 
-There are many ways of contributing, from testing and reporting an issue to suggesting and coding full components or features. Below is a summary of some of the best ways to get involved. 
+There are many forms of contributions. New components or features, changes to existing features, tests, documentation, bug fixes, or just good suggestions. Below is a summary of some of the best ways to get involved.
 
-### Joining Feature Discussions 💭
-
-You can add your voice to discussions around new and existing component features by commenting on an RTC. New components and features that openly invite public comment are marked by an [RTC](https://github.com/adobe/aem-core-wcm-components/labels/rtc) (Request to Comment) label. We strongly encourage users of the Core Components to bring their own project experience to these issues, as there may be alternative use-cases or requirements that haven't yet been considered.
+The Core Component Engineering Team reviews all issues and contributions submitted by the community. During the review we might require clarifications from the contributor. If there is no response from the contributor within two weeks time, the issue will be closed.
 
-### Joining Developer Discussions 💬
+Once a contribution is being reviewed, the Core Engineering Team will apply a relevant label. You can see our [label list on GitHub](https://github.com/adobe/aem-core-wcm-components/labels) to better understand what each label means..
```

Comment:
we should perhaps make it clear here that the label is added to the associated issue, rather than the pull request/contribution? and that all pull requests require an issue to be considered.